src/format.d.ts

```diff
@@ -17,11 +18,12 @@ export function formatMonth(
  * Returns a function that formats a given week day number (from 0 = Sunday to 6
  * = Saturday) according to the specified *locale* and *format*.
  *
- * @param locale a [BCP 47 language tag](https://tools.ietf.org/html/bcp47);
- * defaults to U.S. English.
- * @param format a [weekday
- * format](https://tc39.es/ecma402/#datetimeformat-objects): either *narrow*,
- * *short*, or *long*; defaults to *short*.
+ * @param locale a [BCP 47 language tag][1]; defaults to U.S. English.
+ * @param format a [weekday format][2]: either *narrow*, *short*, or *long*;
+ * defaults to *short*.
```

Comment:
Aside: I think we could drop the `either *narrow*, *short*, or *long*` here because that’s already clear from the type.